src/components/Home.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+const Home = () => (
+  <div className="home">
+    <h3>Welcome to Our Page</h3>
+    <p>
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam illum
+      laudantium aspernatur porro dolorum molestias quaerat quidem praesentium.
+      At sed temporibus perspiciatis aut repellendus eos, iste animi asperiores
+      debitis optio reiciendis illum consectetur aspernatur blanditiis tempore
+      quas placeat officiis nihil recusandae voluptatem? Quam, quidem illo
+      recusandae veritatis itaque dolorum ea impedit ex sit veniam dolore,
+      corrupti esse non nulla velit, cumque quas atque. Est aut temporibus earum
+      voluptates qui itaque? Accusantium, quod. Molestias dolor architecto,
+      assumenda corporis quia, tempora quis beatae sequi eligendi quibusdam
+      itaque culpa. Soluta earum corporis ratione magni reprehenderit accusamus
+      quos iure, est, quidem, aliquid a rem quasi delectus iusto veniam! Placeat
+      a sint illum, fugiat corporis cum quos veniam officia minus error iure
+      ipsa non quasi laboriosam numquam dolores, voluptatibus eveniet velit!
+      Impedit dolores mollitia quae molestiae dolorum, praesentium autem aliquid
+      similique voluptas eos consequuntur architecto libero eveniet dolore
+      reiciendis consectetur perferendis nihil laudantium minus iure? Suscipit
+      quae nam voluptas dolor, repellat ut illo ex minima? Deleniti porro nam
+      iusto tempora et soluta ad sapiente, repudiandae possimus vitae tenetur
+      atque necessitatibus ullam reiciendis at id laborum. Culpa obcaecati iure
+      error debitis! Repudiandae aut blanditiis odio. Sed dolore reiciendis
+      laboriosam odit. Rem sed quis odio veritatis distinctio?
+    </p>
+  </div>
+);
```

Comment:
- [ Required ] I think it would be better if you fix this error from `react`, currently, when I run your application I encounter the error below. I would like to suggest that you change the imports in this file to use `single` quotes and not `double` quotes as shown below.

```JavaScript
import React from 'react';

const Home = () => (
  <div className="home">
    <h3>Welcome to Our Page</h3>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Aperiam illum
      laudantium aspernatur porro dolorum molestias quaerat quidem praesentium.
      At sed temporibus perspiciatis aut repellendus eos, iste animi asperiores
      debitis optio reiciendis illum consectetur aspernatur blanditiis tempore
      quas placeat officiis nihil recusandae voluptatem? Quam, quidem illo
      recusandae veritatis itaque dolorum ea impedit ex sit veniam dolore,
      corrupti esse non nulla velit, cumque quas atque. Est aut temporibus earum
      voluptates qui itaque? Accusantium, quod. Molestias dolor architecto,
      assumenda corporis quia, tempora quis beatae sequi eligendi quibusdam
      itaque culpa. Soluta earum corporis ratione magni reprehenderit accusamus
      quos iure, est, quidem, aliquid a rem quasi delectus iusto veniam! Placeat
      a sint illum, fugiat corporis cum quos veniam officia minus error iure
      ipsa non quasi laboriosam numquam dolores, voluptatibus eveniet velit!
      Impedit dolores mollitia quae molestiae dolorum, praesentium autem aliquid
      similique voluptas eos consequuntur architecto libero eveniet dolore
      reiciendis consectetur perferendis nihil laudantium minus iure? Suscipit
      quae nam voluptas dolor, repellat ut illo ex minima? Deleniti porro nam
      iusto tempora et soluta ad sapiente, repudiandae possimus vitae tenetur
      atque necessitatibus ullam reiciendis at id laborum. Culpa obcaecati iure
      error debitis! Repudiandae aut blanditiis odio. Sed dolore reiciendis
      laboriosam odit. Rem sed quis odio veritatis distinctio?
    </p>
  </div>
);
export default Home;
```
## Screenshot Error
![Screenshot from 2023-07-22 03-15-08](https://github.com/Samuelwanza/mathmagicians/assets/16392046/b3a54312-3c3c-43ff-8b23-850c8a28915a)